silverback/runner.py

```diff
@@ -46,6 +46,15 @@ async def _event_task(
         """
 
     async def run(self):
+        """
+        Run the task broker client for the assembled `SilverbackApp` application.
```

Comment:
note: single ticks are for cursive in method docs. These are not markdown!

Reply:
```suggestion
        Run the task broker client for the assembled ``SilverbackApp`` application.
```